src/app/app.js

```diff
@@ -22,7 +22,13 @@
     'ui.codemirror',
     'hljs',
     'ngSanitize'
-  ]).config(['hljsServiceProvider', function (hljsServiceProvider) {
+  ]).config(['$locationProvider', 'hljsServiceProvider', function ($locationProvider, hljsServiceProvider) {
+
+    $locationProvider.html5Mode({
+      enabled: true,
+      requireBase: false
+    });
```

Comment:
Is it required to have html5Mode location provider? This could conflict with navigation inside other applications that embed the console